### PR TITLE
Feature/mds 3

### PR DESCRIPTION
**Pull request by AdamMc331**

### CODE REVIEW

File: lib/src/main/res/layout/list_item_color.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (AdamMc331) [inline lib/src/main/res/layout/list_item_color.xml]: Note for @Mauker1 : I added this TextView to display the color name, but I realized we don't have a good way to get that yet. I'm going to make another ticket for this in the future. In the mean time, it's empty and won't affect anything.

**Comment by AdamMc331** [comment]
Before we can merge this, I'll need to update the PR to include a way for the user to pass what colors/levels should appear in the picker.

File: lib/src/main/res/layout/list_item_color.xml (hunk 1 above)
Comment (AdamMc331) [inline lib/src/main/res/layout/list_item_color.xml]: https://github.com/androidessence/MaterialDesignSpecs/issues/10

Comment (Mauker1) [inline lib/src/main/res/layout/list_item_color.xml]: Yeah. I kinda have snippet that can calculate the contrast and change the text color accordingly. We could try that.

Comment (Mauker1) [inline lib/src/main/res/layout/list_item_color.xml]: Found the gist: https://gist.github.com/Mauker1/575e048fb2ee21856c35760babc35b3e

Comment (AdamMc331) [inline lib/src/main/res/layout/list_item_color.xml]: Good call. I would add that as a comment on the #10 ticket. I don't think it's necessary for this PR.

**Pull request merged**